src/switch_exporter/server.py

```diff
@@ -64,6 +64,9 @@ def get_arguments() -> argparse.Namespace:
     parser.add_argument(
         '--password', default='monitor',
         help='Password on switches')
+    parser.add_argument(
+        '--keyfile',
+        help='SSH key for switches')
```

Comment:
```suggestion
        help='SSH client key for switches')
```
(just to disambiguate it from the host key)